odh/base/jupyterhub/notebook-images/audio-decoder-demo.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: audio-decoder-notebook
+  labels:
+    opendatahub.io/notebook-image: "true"
```

Comment:
@Gkrumbach07 would you mind filling in some additional `annotations` for us please?

Like this:
https://github.com/operate-first/apps/blob/e3277f3e24ef28502d0d84ea62cead775c4cba74/odh/base/jupyterhub/notebook-images/categorical-encoding.yaml#L7-L13